### DISCUSSION TITLE
Transfer to iTunes on PC?

### DISCUSSION THREAD

**Discussion by GoddessGirrl** [openaudible/openaudible#578] [Q&A] [answered]
I'm brand new to OpenAudible and came here because the company won't let me download the books I paid for in they way I want to download them. Will openaudible allow me to DL my books and put them in iTunes? Thank you for your help!

**Comment by openaudible** [comment, marked as answer]
Once you've converted the files to m4b, just drag and drop them onto iTunes and they should go right in. If you convert to mp3, its more of a pain because you have to tell iTunes that they're audiobooks and not music files. But you should also check out Apple's Books app if you're on a mac. But haven't spent much time with either itunes or books.
